### PR TITLE
Vp/feature wp cli doc

### PR DESCRIPTION
**Pull request by pvaishnaviprdxn**

### CODE REVIEW

File: docs/decision-records/wp-cli.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+id: CLI
```
Comment (yvipinaxioned) [inline docs/decision-records/wp-cli.md:2]: Can we use the official name as `WP-CLI` everywhere?

**Review by yvipinaxioned — CHANGES_REQUESTED** [review]

File: docs/decision-records/wp-cli.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+- PHP 5.6 or later.
+- WordPress 3.7 or later.
+
+## Installation of WP-CLI
+
+**Ubuntu:**
+
+Run the below commands in terminal:
+
+**For Linux:**
```
Comment (smehboobprdxn) [inline docs/decision-records/wp-cli.md:27]: for Linux and Ubuntu it should be same if so can we club its contradicting (looks like missing command for ubuntu)

File: docs/decision-records/wp-cli.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
+---
+id: WP-CLI
+title: WP-CLI
+sidebar_label: WP-CLI
+tags: cli, wp-cli, maintenance, security, updates, user-roles, plugin-updates
+---
+
+This document is an overview of the required information that a developer needs to have in order to use WP-CLI in any of the WordPress projects.
+
+# Overview
```
Comment (smehboobprdxn) [inline docs/decision-records/wp-cli.md:10]: need to revisit the heading hierarchy

File: docs/decision-records/wp-cli.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+```
+wp core verify-checksums --version=$(wp core version)
+```
+
+Plugins security check:
+```
+wp plugin verify-checksums --all
+```
+
```
Comment (smehboobprdxn) [inline docs/decision-records/wp-cli.md:177]: The custom command topic is not covered which we used for installing the setup in our WordPress starter repo

can you please add about that how we can create and use custom CLI commands

File: docs/decision-records/wp-cli.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+This document is an overview of the required information that a developer needs to have in order to use WP-CLI in any of the WordPress projects.
+
+# Overview
+
+WP-CLI is a command line interface which allows the users to manage their WordPress websites from the command prompt. To upgrade and install plugins/themes, to generate backups, new posts can be published and most of the regular admin actions can be performed with a set of commands.
+
+The beauty of the WP-CLI is that it gives us direct control over site. Anything that we do in WordPress admin dashboard can be done using WP-CLI instead. 
+
+**Pre requisites**
+- UNIX-like environment (OS X, Linux, FreeBSD, Cygwin); limited support in Windows
```
Comment (yvipinaxioned) [inline docs/decision-records/wp-cli.md:17]: `limited support in Windows` can we list down things that aren't supported or attach any reference links? 
And if we are confident that our use cases are covered then maybe we can remove this line?

**Review by yvipinaxioned — CHANGES_REQUESTED** [review]

**Review by smehboobprdxn — APPROVED** [review]

**Review by smehboobprdxn — APPROVED** [review]

**Review by yvipinaxioned — APPROVED** [review]

**Pull request merged**